versioned_docs/version-1.0.0/install-kubeslice/kubeslice-cli/topology-configuration.mdx

```diff
@@ -85,6 +87,34 @@ The following table describes the worker chart configuration parameters used to
 | version    | Numeric        | The version of the chart. Leave this value blank for the latest version.         | Optional  |
 | values     | String         | The additional value that can be passed as --set arguments for **helm install**. | Optional  |
 
+## UI Chart Configuration Parameters
+The following table describes the UI chart configuration parameters used to create topology YAML.
+
+| Parameter  | Parameter Type | Description                                                                      | Required  |
+| ---------- | -------------- | -------------------------------------------------------------------------------- | --------- |
+| chart_name | String         | The name of the UI or enterprise chart.                                          | Mandatory |
+| version    | Numeric        | The version of the chart. Leave it blank for the latest version.                 | Optional  |
+| values     | String         | The additional value that can be passed as --set arguments for **helm install**. | Optional  |
+
+## Prometheus Chart Configuration parameters
+The following table describes the Prometheus chart configuration parameters used to create topology YAML.
+
+| Parameter  | Parameter Type | Description                                                                      | Required  |
+| ---------- | -------------- | -------------------------------------------------------------------------------- | --------- |
+| chart_name | String         | The name of the Prometheus chart.                                                | Mandatory |
+| version    | Numeric        | The version of the chart. Leave it blank for the latest version.                 | Optional  |
+| values     | String         | The additional value that can be passed as --set arguments for **helm install**. | Optional  |
+
+## Image Pull Secret Configuration Parameters
+The following table describes the image pull secret configuration parameters used to create topology YAML.
```

Comment:
Add this as a second sentence after topology YAML. 

`These parameters are only required when you use the enterprise-demo option. `